client/src/plus/index.scss

```diff
@@ -68,6 +72,7 @@
           border: none;
           font-size: 20px;
           padding: 7px 15px 9px;
+          z-index: 10001;
```

Comment:
We have a utility that you can use here `$bring-to-front` - https://github.com/mdn/mdn-minimalist/blob/0ec61f3e48a2539297e54b2f7363df5b76f282e9/sass/vars/_layout.scss#L41